pythonx/jedi_vim.py

```diff
@@ -444,6 +444,7 @@ def show_goto_multi_results(definitions, mode):
             # well.
             lst.append(dict(text=PythonToVimStr(d.description)))
         else:
+            # EL: Error here
```

Comment:
What does this mean?

Reply:
That was where I thought the bug I was referring to in the description above was coming from, but I narrowed it down to this: https://github.com/davidhalter/jedi-vim/blob/master/autoload/jedi.vim#L484

Reply:
Still have no clue why an error is being emitted 😬